src/backend/web/routes/admin.js

```diff
@@ -2,22 +2,32 @@ require('../../lib/config');
 const express = require('express');
 const { UI } = require('bull-board');
 const fs = require('fs');
+const { logger } = require('../../utils/logger');
 
 const router = express.Router();
 
 router.use('/queues', UI);
 
 router.get('/log', (req, res) => {
+  let readStream;
   if (!process.env.LOG_FILE) {
     res.send('LOG_FILE undefined in .env file');
   } else {
     try {
-      console.log("I'm trying");
-      const readStream = fs.createReadStream(process.env.LOG_FILE, 'utf8');
+      readStream = fs.createReadStream(process.env.LOG_FILE, 'utf8');
```

Comment:
I'm not sure that you need to open this as `utf8` since you'll stream it over the response, and it can deal with it.